x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/components/fleet_extensions/aws_credentials_form/aws_account_type_selector.test.tsx

```diff
@@ -95,23 +100,15 @@ describe('AwsAccountTypeSelect', () => {
     );
   });
 
-  it('disables AWS Organization radio if awsOrganizationEnabled is false', () => {
-    (useCloudSetup as jest.Mock).mockReturnValue({
-      awsOrganizationEnabled: false,
-      awsPolicyType: 'aws',
-    });
-    renderComponent({
-      packageInfo: { version: '3.0.0' },
-    });
-    const orgRadio = screen.getByLabelText('AWS Organization');
-    expect(orgRadio).toBeDisabled();
+  it('displays correct descriptions based on selected account type', () => {
+    // Single account description (default)
+    renderComponent();
     expect(
-      screen.getByText(/AWS Organization not supported in current integration version/)
+      screen.getByText(/Deploying to a single account is suitable for an initial POC/)
```

Comment:
checking for text is not the best practice, should we add a test id to the element instead? otherwise if wording changes the test breaks which is not the intention I think

Reply:
discussed in Slack, `react-testing-library` actually suggests to prio checking text to testIds as that's how users iterate with the page. Though the brittleness of the tests with this approach is a real thing, I'd personally like to see more scalable approach. eg. checking for a specific substring which is less likely to change without the change in the logic